apps/voting/contracts/test/mocks/VotingMock.sol

```diff
@@ -0,0 +1,20 @@
+pragma solidity 0.4.24;
+
+import "../../Voting.sol";
+
+
+contract VotingMock is Voting {
+    /** Ugly hack to work around this issue:
+     * https://github.com/trufflesuite/truffle/issues/569
+     * https://github.com/trufflesuite/truffle/issues/737
+     */
+    function newVoteWithCast(bytes _executionScript, string _metadata, bool _castVote) auth(CREATE_VOTES_ROLE) external returns (uint256 voteId) {
```

Comment:
For some odd reason, the newer version of truffle only exposes `newVote(bytes, string)` rather than the old version which only exposed `newVote(bytes, string, bool)`.